crowdloans/projects/polkadot.json

```diff
@@ -704,5 +704,29 @@
     "icon": "https://auction-us-east-2.s3.us-east-2.amazonaws.com/icons/chains/aventus.png",
     "website": "https://www.aventus.io/",
     "overview": "Aventus is an enterprise-grade blockchain that marries the scale of a permissioned blockchain with the security and interoperability of public blockchains — with none of the drawbacks of either world.Founded in 2016, Aventus makes building on Polkadot, Ethereum and beyond accessible for businesses and decentralised platforms, bringing interoperability, scalability and sustainability to Ethereum transactions. In its first year of operation, the Aventus Network has onboarded over 20 million transactions for business clients with 2 million active wallets.Aventus’ mission is to enable companies in any ecosystem to capitalise on a vast series of use cases – including NFTs, video games, loyalty schemes, event ticketing, supply chain provenance, and cargo management – via enterprise-grade modular architecture and future-proof themselves via blockchain interoperability."
+  },
+  {
+    "paraid": "2090",
+    "name": "Oak Network",
+    "token": "OAK",
+    "description": "Automation Blockchain for Polkadot",
+    "overview": "Oak is an automation blockchain for Defi and recurring payments across protocols. Oak aims to be the best-in-class automation utility across protocols, offering on-chain automation not possible today.",
```

Comment:
`OAK delivers trustless automation, dedicating its block space and optimizing gas to store, trigger, and execute transactions on connected blockchains.`